config/resources/dossier-domain.lisp

```diff
@@ -24,22 +24,19 @@
   :on-path "case-types")
 
 (define-resource subcase ()
-  :class (s-prefix "dbpedia:UnitOfWork")
+  :class (s-prefix "dossier:Procedurestap")
   :properties `((:short-title         :string ,(s-prefix "dct:alternative"))
                 (:title               :string ,(s-prefix "dct:title"))
-                (:subcase-identifier  :string ,(s-prefix "ext:procedurestapNummer"))
                 (:is-archived         :boolean   ,(s-prefix "ext:isProcedurestapGearchiveerd"))
                 (:confidential        :boolean   ,(s-prefix "ext:vertrouwelijk"))
                 (:subcase-name        :string ,(s-prefix "ext:procedurestapNaam"))
                 (:created             :datetime ,(s-prefix "dct:created"))
                 (:modified            :datetime ,(s-prefix "ext:modified"))
-                (:concluded           :boolean ,(s-prefix "besluitvorming:besloten"))
                 (:show-as-remark      :boolean ,(s-prefix "ext:wordtGetoondAlsMededeling"))
-                (:freeze-access-level :boolean ,(s-prefix "ext:freezeAccessLevel"))) ;; deprecated
   :has-one `((case                    :via ,(s-prefix "dossier:doorloopt")
                                       :inverse t
                                       :as "case")
-             (meeting                 :via ,(s-prefix "besluitvorming:isAangevraagdVoor")
+             (meeting                 :via ,(s-prefix "besluitvorming:isAangevraagdVoor") # todo hernoemen naar ext:isAangevraagdVoor
```

Comment:
Mag mee in deze story aangepakt worden? Of is deze elders gepland?

Reply:
Ik zou die meepakken hier + services bekijken

Reply:
@ValenberghsSven  alle services afgaan dan ?